docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Recommendation/MatrixFactorization.cs

```diff
@@ -74,32 +92,44 @@ private static List<MatrixElement> GenerateMatrix()
             var dataMatrix = new List<MatrixElement>();
             for (uint i = 0; i < MatrixColumnCount; ++i)
                 for (uint j = 0; j < MatrixRowCount; ++j)
-                    dataMatrix.Add(new MatrixElement() { MatrixColumnIndex = i, MatrixRowIndex = j, Value = (i + j) % 5 });
+                    dataMatrix.Add(new MatrixElement() { MatrixColumnIndex = i,
+                        MatrixRowIndex = j, Value = (i + j) % 5 });
+
             return dataMatrix;
         }
 
-        // A class used to define a matrix element and capture its prediction result.
+        // A class used to define a matrix element and capture its prediction
+        // result.
         private class MatrixElement
         {
-            // Matrix column index. Its allowed range is from 0 to MatrixColumnCount - 1.
+            // Matrix column index. Its allowed range is from 0 to
+            // MatrixColumnCount - 1.
             [KeyType(MatrixColumnCount)]
             public uint MatrixColumnIndex { get; set; }
             // Matrix row index. Its allowed range is from 0 to MatrixRowCount - 1.
             [KeyType(MatrixRowCount)]
             public uint MatrixRowIndex { get; set; }
-            // The actual value at the MatrixColumnIndex-th column and the MatrixRowIndex-th row.
+            // The actual value at the MatrixColumnIndex-th column and the
+            // MatrixRowIndex-th row.
             public float Value { get; set; }
-            // The predicted value at the MatrixColumnIndex-th column and the MatrixRowIndex-th row.
+            // The predicted value at the MatrixColumnIndex-th column and the
+            // MatrixRowIndex-th row.
             public float Score { get; set; }
         }
 
         // Print some evaluation metrics to regression problems.
         private static void PrintMetrics(RegressionMetrics metrics)
         {
-            Console.WriteLine($"Mean Absolute Error: {metrics.MeanAbsoluteError:F2}");
-            Console.WriteLine($"Mean Squared Error: {metrics.MeanSquaredError:F2}");
-            Console.WriteLine($"Root Mean Squared Error: {metrics.RootMeanSquaredError:F2}");
-            Console.WriteLine($"RSquared: {metrics.RSquared:F2}");
+            Console.WriteLine("Mean Absolute Error: " + metrics.MeanAbsoluteError +
+                ":F2");
```

Comment:
Is this correct? I thought `F2` tells the prcision.

Reply:
I'm not sure either. I tried reformatting it like another example Zeeshan helped us with that used a similar format. Do you know what the right notation is?